Adjustments to the protocols/generator for `Fleur` code

janssenhenning opened the pull request:
This PR adds the `verification-pbe-v1` protocol and a couple of bugfixes to enable calculations with `aiida-fleur`.
In addition the output remote-folder is added to the `FleurCommonRelaxWorkChain`

janssenhenning commented:
@bosonie This PR is ready and is the version used for the final results of Fleur.
The only exception is that the TS contribution was added to the aiida-fleur plugin output after obtaining the final results. But subsequent calculations with this version will have this available also

bosonie commented:
@janssenhenning so there is no way to take a calculation already run for the verification project and extract the TS value. Is it correct? Just to confirm

janssenhenning commented:
@bosonie The TS contribution is in the output file of fleur also for the already run calculations. It is just not extracted into the output dictionary of the `FleurCalculation` in `aiida-fleur`. So in principle the code for extracting this from the output file directly could be duplicated in the extractor function to also get the contribution for the already run calculations.

bosonie commented:
@janssenhenning. the solution you suggest would be ideal. Would you mid to implement that in an `if`/`try` block? Meaning that the standard behavior  is to parse from the output dictionary of Fleur, but if not found we fall down to extract TS from the output file directly?
Thanks!

janssenhenning commented:
@bosonie I added the fallback solution now. Since I added some explicit usages of `masci-tools` (This a library from our institute, which contains all the concrete Fleur parsing logic) I added this to the dependencies

bosonie commented:
@janssenhenning great! Thanks!
